easybuild/easyconfigs/g/gmsh/gmsh-4.8.4-foss-2020b-Python-3.8.6.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.8.4'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@Falkor Please drop the `versionsuffix`, no longer needed for Python 3 in the `2020b` generation of easyconfigs

Reply:
@boegel Thanks for the comments, I corrected accordingly. Shall I rename also the EB to `gmsh-4.8.4-foss-2020b.eb` ?

Reply:
my bad, big mess and mistake in the EB file to correct -- sry about that, I'll correct it asap